tests/Testing/Temporary/CTestCostData.txt

```diff
@@ -0,0 +1 @@
+---
```

Comment:
I think you didn't want this to be part of the pull request. :slightly_smiling_face:

Reply:
can I just removed the two temporary files instead of going back to a earlier commit? Sorry, that I touched the files.

Reply:
Yes, just delete the files in another commit. Also, run the indentation script at some point (again).